guides/creating-a-plugin.md

```diff
@@ -1,3 +1,318 @@
 # Creating a Feathers Plugin
 
-We haven't had time to publish this guide yet. If you feel like you could contribute one feel free to [open a pull request](https://github.com/feathersjs/feathers-docs/edit/master/guides/creating-a-plugin.md).
\ No newline at end of file
+The easiest way to create a plugin is with the [Yeoman generator](https://github.com/feathersjs/generator-feathers-plugin).
+
+First install the generator
+```js
+$ npm install -g generator-feathers-plugin
+
+```
+
+Then in a new directory run: 
+```js
+$ yo feathers-plugin
+```
+
+This will scaffold out everything that is needed to start writing your plugin. 
+
+Output files from generator: 
+```
+   create package.json
+   create .babelrc
+   create .editorconfig
+   create .jshintrc
+   create .travis.yml
+   create example/app.js
+   create src/index.js
+   create test/index.test.js
+   create README.md
+   create LICENSE
+   create .gitignore
+   create .npmignore
+```
+
+Simple right? We technically could call it a day as we have created a Plugin. However, we probably want to do just a bit more. Generally speaking a Plugin is a [Service](../services/readme.md). The fun part is that a Plugin can contain multiple Services which we will see below. 
+
```

Comment:
It would be nice at this point to explain what the tutorial services will be, so readers have an idea of the functionality that they will be building.